### PR TITLE
Brain result

### PR DESCRIPTION
**Pull request by ForJadeForest**
This PR is to show the result of orca brainMRI. 
- add the result to readme. 
- add the validation dataset when training and test dataset
- modify the metrics calculation function. 
- add the scheduler. 
- modify the `torch_distributed` backend to `ray` backend. 
- add the brainMRI to the test script.

### CODE REVIEW

File: python/orca/dev/example/run-example-test-ray.sh [hunk 1]
```diff
@@ -178,4 +178,4 @@ echo "#9 fashion-mnist time used:$time9 seconds"
 echo "#10 orca super-resolution example time used:$time10 seconds"
 echo "#11 orca cifar10 example time used:$time11 seconds"
 echo "#12 auto-xgboost-regressor-spark-df example time used:$time12 seconds"
-echo "#13 ray-dataset-xgboost example time used:$time13 seconds"
+echo "#13 ray-dataset-xgboost example time used:$time13 seconds"
```
Comment (hkvision) [inline python/orca/dev/example/run-example-test-ray.sh:181]: revert this change

File: python/orca/example/learn/pytorch/brainMRI/README.md [hunk 2]
```diff
@@ -64,9 +64,19 @@ python brainMRI.py --cluster_mode yarn-client --backend spark --model_dir hdfs_p
...
 - `--epochs` The number of epochs to train for. Default is 2.
 - `--batch_size` The number of samples per gradient update. Default is 64.
 - `--data_dir` The path to the dataset. Default is `./kaggle_3m`.
 - `--additional_archive` The zip dataset if use `yarn-client` mode. Default is `kaggle_3m.zip#kaggle_3m`.
-- `--model_dir` The model save dir when use spark backend. Default is the current working directory.
\ No newline at end of file
+- `--model_dir` The model save dir when use spark backend. Default is the current working directory.
+
+## Results
+**For "ray" and "spark" backend**
```
Comment (hkvision) [inline python/orca/example/learn/pytorch/brainMRI/README.md:75]: backend -> backends

**Review by hkvision — APPROVED** [review]
